server/models/create.js

```diff
@@ -0,0 +1,119 @@
+import bcrypt from 'bcryptjs';
+import dotenv from 'dotenv';
+import db from './dbConnect';
+
+dotenv.config();
+
+const hashedPassword = bcrypt.hashSync(process.env.JWT_SECRET, 10);
+const userSeeding = `
+DROP TABLE IF EXISTS users CASCADE;
+DROP TYPE IF EXISTS status;
+CREATE TYPE status AS ENUM('user','admin');
+CREATE TABLE users(id SERIAL PRIMARY KEY,
```

Comment:
Nice implementation, why not have different files for database creation and seeding of data? Separation of concern since it looks to me that this file is doing just too much.

Reply:
Or better still extract the models into their respective files and create an index.js file where you import all the model creation functions and call them by running that file and a separate seed data file where you do your inserts.

Reply:
Thanks for the feedback. I will implement your first suggestion.